ServerWhitelist.yml

```diff
@@ -1940,3 +1940,16 @@ AllowedPrefixes:
     # By caco on behalf of MissCorruption, Masterstroke dev
     - https://subscribestar.adult/posts/905582 # BakaFactory SLAL Animation 6.41V LE / SE
     - https://subscribestar.adult/posts/993525 # Baka Motion Pack
+
+     # N.Y.A NSFW Skyrim
+    - https://www.mediafire.com/file/xcrzwu3757co1pm/Automated_SLSB_Conversions_0.6c.7z/file # Animation Pack Conversion by BabyImpala with Permission
+    - https://www.mediafire.com/file/j386qo0w4qjm1b3/More_Nasty_Critters_Animations_SE_V15_4.7z/file # Merged Version of all 3 SplitVolumnes by MoreNastyCritters
+    - https://cdn.discordapp.com/attachments/1063514401375780874/1198357469903204383/SexLab_Framework_PPLUS_-_V2.3.2.1_-_1.5.97.zip?ex=65da4bdd&is=65c7d6dd&hm=3121fe0ab2594a7a661e07a0d19c0dafe7f72813946e2be02479e04977e51a9b& # Sexlab + 2.3.2.1 / SE 1.5.9.7 ( Inline Permission from Scrab )
+    - http://www.mediafire.com/file/o5yacfkx21p3b8i/Sexlab_DLL_16640_Replacer.rar # Sexlab Framework 16640 Working DLL ( Inline Permission from Scrab )
+    - https://cdn.discordapp.com/attachments/1177560192192352336/1178355556113731604/SexLab_P_Quick_Scene_Start_Spell_-_4.0.0.zip?ex=65db5ba1&is=65c8e6a1&hm=0afebe8fd8aedc5ce84171cc1b0d649fd62bbe888f4d75c89f95fcdb6c34ab4c& # Sexlab + Quick Spell Inline
+    - https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/releases # Pandora Behaviour Engine v1.1.6-alpha
```

Comment:
github links don't need whitelisting so this line should be removed

Reply:
Will remove the Github one.